gennav/planners/graph_search/astar.py

```diff
@@ -17,7 +21,9 @@ def __lt__(self, other):
 
     # Compare nodes
     def __eq__(self, other):
```

Comment:
Is this necessary if you have defined `__eq__` in `Node` also

Reply:
every code that uses node may not be require the same eq. Can be done that way if you want?